build_docs.py

```diff
@@ -257,7 +259,7 @@ class Language:
     Language("fr", "fr", "French", True, XELATEX_WITH_FONTSPEC),
     Language("id", "id", "Indonesian", False, XELATEX_DEFAULT),
     Language("it", "it", "Italian", False, XELATEX_DEFAULT),
-    Language("ja", "ja", "Japanese", True, PLATEX_DEFAULT, html_only=True),  # See https://github.com/python/python-docs-ja/issues/35
+    Language("ja", "ja", "Japanese", True, PLATEX_DEFAULT, html_only=True),
```

Comment:
```suggestion
    Language("ja", "ja", "Japanese", True, LUALATEX_DEFAULT, html_only=True),
```

Also later in the file there's a piece of code that prints the platex version, that should be updated to print the lualatex version instead if platex is no longer used anywhere.